triagebot.toml

```diff
@@ -468,6 +468,9 @@ This was probably unintentional and should be reverted before this PR is merged.
 If this was intentional then you can ignore this comment.
 """
 
+[mentions."src/tools/x/src/main.rs"]
+message = "`src/tools/x/src/main.rs` was changed. Bump version of Cargo.toml in `src/tools/x` ."
```

Comment:
```suggestion
message = "`src/tools/x` was changed. Bump version of Cargo.toml in `src/tools/x` so tidy will suggest installing the new version.
```

Reply:
Oh sorry, I meant to suggest changing the trigger file too, any change in src/tools/x should prompt this.